app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision/state_machine.rb

```diff
@@ -42,10 +42,10 @@ def determine_placement
 
   def start_clone_task
     update_and_notify_parent(:message => "Starting Clone of #{clone_direction}")
-
     clone_options = prepare_for_clone_task
     log_clone_options(clone_options)
-    phase_context[:clone_task_mor] = start_clone(clone_options)
+    phase_context[:clone_task_ref] = start_clone(clone_options)
+    phase_context.delete(:clone_options)
```

Comment:
Why delete the clone options here?

Reply:
Just following what's in the base module:
https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/cloud_manager/provision/state_machine.rb#L38